tests/js/setup-globals.js

```diff
@@ -85,3 +85,7 @@ global.PointerEvent = function () {};
 
 // Provide a stub for scrollTo, as it's not implemented by JSDOM. See https://github.com/jsdom/jsdom/pull/2626.
 global.scrollTo = () => {};
+
+// Provide the desktop viewport as default because SK is not mobile-first app.
```

Comment:
```suggestion
// Provide the desktop viewport as default because SK is not mobile-first app. Global dimensions are used in the useWindowSize hook.
```

Reply:
Addressed [here](https://github.com/google/site-kit-wp/pull/10114/commits/32fe4de2bbdab6fdeac6f85efd1c851931144afc)